smokey/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -158,57 +171,88 @@ else if (this.subsystems.getEveryBotPickerSubsystem() != null) {// cube uses upt
     }
     
     /**
-     * Builds a command list for use in auto routines
-     * @param NodePosition starting position of robot corrosponding to the node. Nodes are numbered from left to right 1- 9 from the drivers perspective
-     * @param Trajectory trajectory to follow out of the community
+     * A method to move the arm to the specified scoring position. 
+     * Caller is responsible for checking that the Arm Subsystem is installed. 
+     * @param height
      * @return command
      */
-    public Command getAutoRoutine (Pose2d NodePosition, Trajectory Trajectory){
+    private Command getArmPositionRoutine (ScoringPosition height){
         SequentialCommandGroup command = new SequentialCommandGroup();
-        command.addCommands(getScoreRoutine(NodePosition));
-        command.addCommands(new DriveTrajectoryCommand(subsystems.getDriveTrainSubsystem(), Trajectory));
+        if (height == ScoringPosition.SCORE_HIGH){
+            command.addCommands(new ArmToLocationCommand(subsystems.getArmSubsystem(), ArmToLocationCommand.ArmLocation.ARM_HIGH_SCORE));
+        }
+        else if (height == ScoringPosition.SCORE_MIDDLE){
+            command.addCommands(new ArmToLocationCommand(subsystems.getArmSubsystem(), ArmToLocationCommand.ArmLocation.ARM_MED_SCORE));
+        }
```

Comment:
So we won't ever try for low?  Seems about right.

Also don't actually need to wrap the ArmToLocationCommand in a SequentialCommandGroup but its only a bit of extra memory.

Reply:
Correct.  We don't need low height in auto.

Reply:
removed SequentialCommandGroup